trees/src/binaryTree.js

```diff
@@ -1,66 +1,77 @@
 class Node {
   constructor(value, left, right) {
     this.value = value;
-    this.left = left; // A Node
-    this.right = right; // A Node
+    this.left = left; 
+    this.right = right; 
   }
 
   preOrder() {
-    //Pre-order: root >> left >> right, easiest
-    // Return a single array
     let traversal = [];
-    traversal.push(this.value); // Root
-
+    traversal.push(this.value); 
     if (this.left) {
-    // Goes Left on tree
       let leftTraversal = this.left.preOrder();
       traversal = traversal.concat(leftTraversal);
     }
-
-    // Goes Right on tree
     if (this.right) {
       let rightTraversal = this.right.preOrder();
       traversal = traversal.concat(rightTraversal);
     }
-
     return traversal;
   }
 
   inOrder() {
-    // In - order: left >> root >> right
     let traversal = [];
-    traversal.push(this.value); // Root
+    traversal.push(this.value); 
     if (this.left) {
-    // Goes Left on tree
       let leftTraversal = this.left.inOrder();
       traversal = [...leftTraversal, ...traversal];
     }
-    // Goes Right on tree
     if (this.right) {
       let rightTraversal = this.right.inOrder();
       traversal = [...traversal, ...rightTraversal];
     }
     return traversal;
   }
 
-
-
   postOrder() {
-    // Post - order: left >> right >> root
     let traversal = [];
-    traversal.push(this.value); // Root
-    // Right
+    traversal.push(this.value); 
     if (this.right) {
       let rightTraversal = this.right.postOrder();
       traversal = [...rightTraversal, ...traversal];
     }
     if (this.left) {
-    // Left
       let leftTraversal = this.left.postOrder();
       traversal = [...leftTraversal, ...traversal];
     }
     return traversal;
   }
+
+  max() {
+    let traversal = [];
+    traversal.push(this.value);
+    if (this.right) {
+      let rightTraversal = this.right.postOrder();
+      traversal = [...rightTraversal, ...traversal];
+    }
+    if (this.left) {
+      let leftTraversal = this.left.postOrder();
+      traversal = [...leftTraversal, ...traversal];
+    }
+    return Math.max(...traversal);
+  }
+}
+class BinaryTree {
+  constructor(root) {
+    this.root = root;
+  }
+  preOrder() {
+    return this.root.preOrder();
+
+  }
+  inOrder() { return this.root.inOrder(); }
+  postOrder() { return this.root.postOrder(); }
+  max() { return this.root.max(); }
 }
 
 module.exports = { Node };
```

Comment:
Add BinaryTree so that way your BST has access to it, and in the test change line 15 ` expect(tree.contains(8)).toBe(true);` to false